### PR TITLE
Fix vale checking

### PR DESCRIPTION
**Pull request by Andy-Grigg**
Try incorporating the fix suggested here https://github.com/errata-ai/vale/issues/706#issuecomment-1771748870 to resolve the odd behavior around vale warnings.

### CODE REVIEW

**Comment by Andy-Grigg** [comment]
The lack of warnings indicate this seems to be successful.

**Comment by Andy-Grigg** [comment]
I spoke too soon, looks like vale is failing:

```
  panic: regexp: Compile(`(?i)(?:Azure|CLI|Cosmos|Docker|Emmet|gRPC|I|Kubernetes|Linux|macOS|Marketplace|MongoDB|REPL|Studio|TypeScript|URLs|Visual|VS|Windows|(?!)ansys|Granta MI|PyGranta|RecordLists|API|Makefile|jupytext)|[\p{N}\p{L}*]+[^\s]*`): error parsing regexp: invalid or unsupported Perl syntax: `(?!`
```

**Comment by Andy-Grigg** [comment]
The regex `[Aa][Nn][Ss][Yy][Ss]` runs, but it still flags up notices about sentence case, even for a single-word title like 'Installation'. This isn't an easy fix, so I'll close it and we can try and solve it later.

**Comment by SMoraisAnsys** [comment]
Did you get any progress on that issue ? I had to postpone testing the fixes in https://github.com/errata-ai/vale/issues/706 and am currently trying them. However this PR was request on the discussion and it would be nice to know if you had any success on your side ?

**Comment by SMoraisAnsys** [comment]
@Andy-Grigg I had some nice result from my side (see https://github.com/ansys/pyansys-dev-guide/pull/371)
You should probably give a try at the new version of `vale` + following https://github.com/errata-ai/vale/issues/706 advices.
Note: you mentioned the regex `[Aa][Nn][Ss][Yy][Ss]`, in that case, following https://vale.sh/docs/topics/vocab/#case-sensitivity, I would recommend typing `(?i)Ansys` instead.
Hope that evrything will get back in track on your side 👍

File: doc/styles/Vocab/ANSYS/accept.txt [hunk 1]
```diff
@@ -1,9 +1,7 @@
-ANSYS
-Ansys
-ansys
 Granta MI
 PyGranta
 RecordLists
 API
 Makefile
 jupytext
+(?i)ansys
```
Comment (ludovicsteinbach) [inline doc/styles/Vocab/ANSYS/accept.txt:7]: Doesn't that make "aNsys", "ansYS", etc valid ?

Comment (Andy-Grigg) [reply doc/styles/Vocab/ANSYS/accept.txt:7]: It does, but according to the vale documentation, having multiple versions of the same string with different casing isn't supported. So we'll just need to keep an eye out for those scenarios.

**Review by ludovicsteinbach — APPROVED** [review]

File: doc/styles/Vocab/ANSYS/accept.txt (hunk 1 above)
Comment (Andy-Grigg) [reply doc/styles/Vocab/ANSYS/accept.txt:7]: If Ansys wasn't sometimes used in all caps, it wouldn't be an issue. But in our case, yes, it's not ideal.

**Pull request merged**